src/utils/test-enzyme.ts

```diff
@@ -4,13 +4,13 @@ import ThemeProvider from '../design-tokens/ThemeProvider';
 
 /* eslint-disable @typescript-eslint/explicit-function-return-type */
 
-const shallowWithTheme = (element: React.ReactElement<any>, ...props): any =>
+const shallowWithTheme = (element: React.ReactElement<any>, ...props: undefined[]): any =>
   shallow(element, {
     wrappingComponent: ThemeProvider,
     ...props,
   });
 
-const mountWithTheme = (element: React.ReactElement<any>, ...props): any =>
+const mountWithTheme = (element: React.ReactElement<any>, ...props: undefined[]): any =>
```

Comment:
Sorry, but I am not sure I agree with this type 🧐

Reply:
Again since this is only used to type a unit test and not called by production code I took the fastest route to satisfy the compiler.

Reply:
Hm..I understand your point. I would do so

![image](https://user-images.githubusercontent.com/29228205/69673145-f71af080-1099-11ea-8865-761ccf5a36c2.png)

....but we have this lint rule

The thing is that if the type is "any" or the build is not satisfied, then we know that we need to right the correct type, but when we introduce, for instance undefined[], we might not see where we should fix in the future... do you understand? so even if it's not production code, I would appreciate to have it correct..otherwise leave as it is